tests/xfstests/install.pm

```diff
@@ -26,6 +26,16 @@ use repo_tools 'add_qa_head_repo';
 
 my $STATUS_LOG  = '/opt/status.log';
 my $VERSION_LOG = '/opt/version.log';
+my $LOCAL_DIR   = '/tmp/xfstests-dev';
+my $IBS_REPO    = 'http://download.suse.de/ibs/home:/yosun:/branches:/QA:/Head/SLE-15-SP2';
```

Comment:
It's better set as a variable than hard code.

Reply:
OK, I'll change as you said.